client/components/vertical-menu/style.scss

```diff
@@ -0,0 +1,33 @@
+.vertical-menu {
+	display: flex;
+	flex-direction: column;
+	max-width: 280px;
+}
+
+.vertical-menu__items {
+	@extend %content-font;
+
+	display: flex;
+	height: 48px;
```

Comment:
I think we should also apply `cursor: pointer;` here, as these are clickable items.